### ISSUE TITLE
[BUG] - AXE-Core test fails if tested with open dropdown of `Multi Select`

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3193]
### JS Framework

all

### Porsche Design System version

3.15.0-rc.1

### On which device(s) you can see the bug?

all

### In which browsers you can see the bug?

all

### Browser version

all

### Steps to reproduce the behavior

1. Go to https://designsystem.porsche.com/v3/components/multi-select/examples
2. Open a MultiSelect
3. Do partial test with AXE DevTools
4. See error: Ensures elements with an ARIA role that require child roles contain them

### Expected behavior

AXE-Core test should be also valid for dropdown of `Multi Select`
Re-enable Axe Core Tests for multi-select

### Additional information

The AXE-Core test fails due to the `role="status"` of the loading message inside of the checkbox component which is a child element of the `role="listbox"` element which is not allowed in this context.

<img width="809" alt="Bildschirmfoto 2024-04-22 um 11 17 11" src="https://github.com/porsche-design-system/porsche-design-system/assets/40788023/222fb648-2b62-48ad-84db-d4e849456b9a">

**A11y tree:**
<img width="844" alt="Bildschirmfoto 2024-04-22 um 11 20 31" src="https://github.com/porsche-design-system/porsche-design-system/assets/40788023/c4074f08-658d-4213-8e32-510b029d63a5">

**Comment by stephanschroeter** [comment]
Maybe it's better to just style a checkbox like indicator instead of using a real checkbox.
